pom.xml

```diff
@@ -30,29 +30,22 @@
 		<url/>
 	</scm>
 	<properties>
+		<caffeine.version>3.2.0</caffeine.version>
+		<io.prometheus.version>1.4.0-SNAPSHOT</io.prometheus.version>
 		<java.version>21</java.version>
-		<spring-shell.version>3.4.0</spring-shell.version>
+		<lombok.version>1.18.30</lombok.version>
 		<snmp4j-version>3.7.1</snmp4j-version>
 		<spring-cloud.version>2024.0.1</spring-cloud.version>
 		<spring-doc.version>2.8.6</spring-doc.version>
+		<spring-shell.version>3.4.0</spring-shell.version>
 	</properties>
-	<repositories>
-		<repository>
-			<id>sonatype-snapshots</id>
-			<url>https://oss.sonatype.org/content/repositories/snapshots</url>
-			<snapshots>
-				<enabled>true</enabled>
-			</snapshots>
-		</repository>
-	</repositories>
 	<build>
 		<plugins>
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<source>21</source>
-					<target>21</target>
+					<release>${java.version}</release>
```

Comment:
Reviewer hint: https://maven.apache.org/components/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html